docs/kyma/docs/036-inst-update.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Update Kyma Installation
+type: Installation
+---
+
+This Installation guide describes steps necessary to update Kyma Installation in local and cluster environments.
+
+## Prerequisites
+
+- Kyma is succesfully installed, Installation CR status is `Installed`
+- Access to the Kyma installation cluster with kubectl tool
+- Docker tools to build Docker images
+- Access to a Docker Registry - only for cluster installation
+
+## Overview
+
+Kyma consists of multiple components, installed as [Helm](https://github.com/helm/helm/tree/master/docs) releases.
+
+Kyma update means introducing changes to these releases - changes in charts, changes in override values or both. Adding new Helm releases is also considered an update of an existing installation.
+
+Some updates might not be possible, because of breaking changes between component versions or dependency conflicts. In that case a component-specific work-around is necessary. Such component-specific custom procedures or work-arounds are out of scope of this document. Similarly, any other custom update-related steps, like data migration, are out of scope.
```

Comment:
```suggestion
> **NOTE:** In case of dependency conflicts or major changes between components versions, some updates may not be possible. Further steps assume that the update of all involved components is not obstructed.
```